src/Pages/juyoungson/Login/LoginS.scss

```
@@ -0,0 +1,209 @@
+$mainColor: #fafafa;
+$mainborderColor: #c9c9c9;
+
+.Login {
+  @font-face {
+        font-family: NanumL;
+        src: url("../../../Styles/NanumSquareRoundL.ttf") format("truetype");
+      }
+  @import url("https://fonts.googleapis.com/css2?family=Lobster&display=swap"); 
+  
+  *{
+    font-family: 'NanumL';
+  }
+  height: 100vh;
+  background-color: $mainColor;
+
+  button {
+    width: 80%;
+    font-weight: 900;
+    font-size: 14px;
+    cursor: pointer;
+  }
+
+  #inputPw,
+  #inputId {
+    width: 80%;
+    background-color: $mainColor;
+    text-indent: 10px;
+  }
+
+  .loginMain {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin-bottom: 25px;
+    height: 85%;

```

전체적으로 css 속성 순서(Box Model Properties (width, height, margin, padding)순서로 수정해주시면 좋을 것 같습니다!

```suggestion
   height: 85%;
    margin-bottom: 25px;
   display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
```

---

새미님 리뷰처럼 css 속성 순서 참고하셔서 반영해주세요~